blog/2015/10/graduate_communities.html

```diff
@@ -0,0 +1,63 @@
+  as a possible solution to the issue of supporting learners 
+  after they've attended one of our workshops:
+</p>
+<!-- end excerpt -->
+<p>
+  "Besides the instructor training bottleneck, 
+  it seems to me that the other huge issue facing Software Carpentry
+  is what happens to learners after they've attended a workshop. 
+  Thinking about my own experience, 
+  I attended the workshop you ran in Melbourne in early 2013 
+  and then 2.5 years later I published my first ever
+  <a href="https://www.authorea.com/users/5641/articles/12197/_show_article">reproducible paper</a>
+  (as I continually have to explain to impatient managers at Research Platform Services, 
+  yes it will take people at least that long to fully work these things into their daily workflow). 
+  The difference between me 
+  and a Software Carpentry attendee who never goes on to adopt the best practices we teach 
+  is probably not that I read better books, attend better follow-up classes or find better online tutorials, 
+  but rather that I'm part of an active community. 
+  In fact, besides that initial Software Carpentry workshop,
+  I can't recall a single book, class or tutorial that was hugely influential in my development as a computational scientist. 
+  Your two-day Software Carpentry workshop in 2013 introduced me to the core concepts I needed to know, 
+  and then I stuck at working to incorporate them into my working life because the people 'around' me, 
+  in both the Software Carpentry instructor community and the 
+  <a href="http://pyaos.johnny-lin.com/">Python in the Atmospheric and Ocean Sciences</a> community, 
+  were also doing it.
+</p>
+<p>
+  My thought for tackling this issue is therefore to insert Software Carpentry graduates 
+  into communities that are programming language and research discipline specific. 
+  In other words,
+  plug them into an environment where doing and discussing computational best practice is the norm, 
+  and where these discussions happen in a language that is most familiar and relevant to them. 
+  In some cases these communities already exist 
+  (e.g. we could ask PyAOS if it would like to be the official community for Software Carpentry graduates 
+  who use Python in the atmospheric and ocean sciences) 
+  and in other cases we could approach people in the Software Carpentry community to help establish them 
+  (e.g. Software Carpentry graduates who do bioinformatics using R). 
+  These official Software Carpentry alumni groups would have a mailing list for general chat and questions 
+  (that people are automatically signed up to after attending a workshop), 
+  a blog, a discipline and language specific capstone lesson that they maintain
+  and they may even host workshops at relevant conferences 
+  (e.g. PyAOS runs a workshop at the American Meteorological Society Annual Meeting every year)."  
+</p>
+<p>
+  What do people think? Is this an initiative that the Steering Committee should look at implementing? 
```

Comment:
> Is this an initiative that the Steering Committee should look at implementing?

This was one of the reasons that we helped NumFOCUS with this year edition of GSoC.